### PR TITLE
spring5 compatibility test module

### PR DESCRIPTION
**Pull request by TheRealMarnes**
Just needs a fix for that weird build error where travis gets hung up on maven downloading xmlunit or spring-something in the mavencentral run

### CODE REVIEW

File: spring5/pom.xml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>jdbi3-parent</artifactId>
+        <groupId>org.jdbi</groupId>
+        <version>3.6.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>jdbi3-spring5</artifactId>
```
Comment (stevenschlansker) [inline spring5/pom.xml]: we probably should not publish this artifact, lest people search on `search.maven.org` and see, "oh, of course I should use `jdbi3-spring5`!"

File: spring5/pom.xml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    <description>jdbi Spring 4/5 forward compatibiility testing</description>
+    <url>http://jdbi.org/</url>
+
+    <properties>
+        <dep.spring.version>5.1.3.RELEASE</dep.spring.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework</groupId>
```
Comment (stevenschlansker) [inline spring5/pom.xml]: compile scope dependency on an artifact with no compiled sources

File: spring5/pom.xml (hunk 1 above)
Comment (TheRealMarnes) [reply spring5/pom.xml]: I thought it wouldn't generate an artifact without any sources to compile into it... :/

**Comment by TheRealMarnes** [comment]
Gave your feedback a try, dunno if it'll make the cut

File: spring5/pom.xml (hunk 1 above)
Comment (TheRealMarnes) [reply spring5/pom.xml]: I suppressed the install and deploy plugins further below, does that resolve this or is there something ekse?

File: spring5/pom.xml (hunk 2 above)
Comment (TheRealMarnes) [reply spring5/pom.xml]: I set it to test scope

**Comment by TheRealMarnes** [comment]
Still seems to be getting that weird build timeout

**Comment by TheRealMarnes** [comment]
Uh, I hope the build works...

**Pull request merged**